### PR TITLE
Fix misc. WOCCatalog breaks in Xcode

### PR DESCRIPTION
**Pull request by s5msft**
Various (small) fixes to parts of WOCCatalog when built for iOS

### CODE REVIEW

File: samples/WOCCatalog/WOCCatalog/PickersViewController.m [hunk 1]
```diff
@@ -23,31 +23,22 @@ - (void)viewDidLoad {
...
+    [self tableView].separatorColor = [UIColor clearColor];
     [self tableView].allowsSelection = NO;
+    [self tableView].scrollEnabled = NO;
 
     self.title = @"Pickers";
     
     // PICKER VIEW
-    
-    self.datePickerView = [[UIDatePicker alloc] initWithFrame:CGRectZero];
+    self.datePickerView = [[UIDatePicker alloc] initWithFrame:[self calcFrameForPicker]];
```
Comment (DHowett-MSFT) [inline samples/WOCCatalog/WOCCatalog/PickersViewController.m]: If we're overriding the size later, there's no point in setting it here.
If there is, something's broken :)

File: samples/WOCCatalog/WOCCatalog/PickersViewController.m [hunk 2]
```diff
@@ -129,20 +95,19 @@ - (void)action:(id)sender
     }
 }
 
-- (CGRect)calcFrameForPicker:(CGSize)size
+- (CGRect)calcFrameForPicker
 {
     CGRect screenRect = [[UIScreen mainScreen] applicationFrame];
```
Comment (DHowett-MSFT) [inline samples/WOCCatalog/WOCCatalog/PickersViewController.m]: This should probably be self.view.bounds -- the view should scale first, and its contents should not be based on screen size

File: samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm [hunk 3]
```diff
@@ -42,7 +42,6 @@ - (void)viewDidLoad {
         [self setupAccelerometer];
     } else {
         accLabel = [[UILabel alloc] initWithFrame:CGRectMake(30, accHeight, 300, 50)];
-        [accLabel setBackgroundColor:nil];
```
Comment (DHowett-MSFT) [inline samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm]: In which colour does nil result? Is it different in iOS versus WinObjC? Should it ever be clear?

File: samples/WOCCatalog/WOCCatalog/PickersViewController.m (hunk 2 above)
Comment (s5msft) [inline samples/WOCCatalog/WOCCatalog/PickersViewController.m]: Several of these were just to maintain some of the existing code. But, fine, I'll update and resubmit.

File: samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm (hunk 3 above)
Comment (s5msft) [inline samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm]: nil is black, and unspecified is clear

**Comment by Ramu-msft** [comment]
@bdrlamb-ms is added to the review. #Closed

**Comment by Ramu-msft** [comment]
@davelamb is added to the review. #Closed

**Comment by Ramu-msft** [comment]
@oliversa-msft is added to the review. #Closed

File: samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm (hunk 3 above)
Comment (bdrlamb) [inline samples/WOCCatalog/WOCCatalog/CoreMotionViewController.mm]: Might want to use [UIColor blackColor] for more clarity

---
In reply to: [64473453](https://github.com/Microsoft/WinObjC/pull/512#discussion_r64473453) [](ancestors = 64473453)

**Comment by DHowett-MSFT** [comment]
:shipit:

**Pull request closed without merging**